tests/test_management.cpp

```diff
@@ -160,6 +161,11 @@ class ProjectTest : public ::testing::Test {
                 std::cout << "PLY[" << i << "] iter number mismatch: " << ply_a.ply_training_iter_number << " vs " << ply_b.ply_training_iter_number << std::endl;
                 return false;
             }
+
+            if (ply_a.ply_name != ply_b.ply_name) {
+                std::cout << "PLY[" << i << "] name mismatch: '" << ply_a.ply_name << "' vs '" << ply_b.ply_name << "'" << std::endl;
```

Comment:
Should be a log

Reply:
tried it but i cant see the log, probably because it is a gtest and should be initialized properly